engine/src/flutter/shell/platform/embedder/embedder.cc

```diff
@@ -1649,10 +1657,14 @@ MakeViewportMetricsFromWindowMetrics(
            "physical height or width.";
   }
 
-  metrics.physical_min_width_constraint = metrics.physical_width;
-  metrics.physical_max_width_constraint = metrics.physical_width;
-  metrics.physical_min_height_constraint = metrics.physical_height;
-  metrics.physical_max_height_constraint = metrics.physical_height;
+  if (metrics.physical_width != 0) {
```

Comment:
I expect that many embedders will update their embedder.h and not realize they need to add constraints to their metrics.

Could we add a `FML_LOG(ERROR)` message if the constraints are all `0` but the `width` and `height` are non-zero? I don't believe there's any valid use cases for this scenario, but this indicates an embedder that has forgotten to pass constraint values.